src/modules/core/components/PermissionsLabel/PermissionsLabel.tsx

```diff
@@ -68,10 +68,10 @@ const PermissionsLabel = ({
     <Tooltip
       content={
         <div className={styles.tooltip}>
-          {infoMessage && infoMessage === 'string' ? (
-            infoMessage
-          ) : (
+          {infoMessage && typeof infoMessage === 'object' ? (
```

Comment:
why did you keep the truthy check on infoMessage? you removed this check on all other 
```
typeof ...  === 'object'
```

Reply:
The rationale for this one is that infomessage could be undefined according to its typing, so I check for it. Whereas on the ones where I removed it, they are not optional according to their typings, therefore I can safely assume they won't be undefined.

Reply:
If you do `typeof undefined` it will be 'undefined' and definitely won't equal `string` or `object` so it's really not necessary

Reply:
I take my words back. It's actually correct. I've misread the code initially